lottie-ios/Classes/Extensions/UIColor+Expanded.m

```diff
@@ -293,6 +305,12 @@ - (NSString *)LOT_stringFromColor {
 		case kCGColorSpaceModelMonochrome:
 			result = [NSString stringWithFormat:@"{%0.3f, %0.3f}", self.white, self.alpha];
 			break;
+        case kCGColorSpaceModelLab:
+        case kCGColorSpaceModelCMYK:
+        case kCGColorSpaceModelDeviceN:
+        case kCGColorSpaceModelIndexed:
+        case kCGColorSpaceModelPattern:
+        case kCGColorSpaceModelUnknown:
 		default:
```

Comment:
That's right, the `default` case could be removed if all the cases are listed explicitly. I'll do that.
Thanks for the feedback!